.azure-pipelines/generation-pipeline.yml

```diff
@@ -508,3 +518,57 @@ stages:
         cleanMetadataFolder: $(cleanMetadataFolderV1)
         languageSpecificSteps:
         - template: generation-templates/objc.yml
+
+- stage: stage_cli_v1
+  dependsOn:
+  - stage_build_and_publish_kiota
+  - stage_v1_openapi
+  condition: |
+    and
+    (
+      eq(dependencies.stage_build_and_publish_kiota.result, 'Succeeded'),
+      eq(dependencies.stage_v1_openapi.result, 'Succeeded')
+    )
+  jobs:
+  - job: cli_v1
+    steps:
+    - template: generation-templates/language-generation-kiota.yml
+      parameters:
+        language: 'shell'
+        version: ''
+        repoName: 'msgraph-cli'
+        branchName: $(v1Branch)
+        targetClassName: "GraphServiceClient"
+        targetNamespace: "github.com/microsoftgraph/msgraph-cli/"
```

Comment:
The namespace has this format for go as this is how things work in go, but you probably want to update that for CLI (dotnet)